### PR TITLE
VCK5000 documentation fix

### PR DESCRIPTION
**Pull request by eddierichter-amd**
Adding some additional functionality to the `build-mlir-aie-pcie.sh` script to automatically build ROCr and aie-rt if install paths aren't provided. Also cleaning up some of the VCK5000 build documentation.

### CODE REVIEW

File: docs/Building.md [hunk 1]
```diff
@@ -118,42 +115,42 @@ and llvm.
     source utils/env_setup.sh <mlir-aie>/install <llvm dir>/install
     ```
 
+6. **Only for the VCK5000:** The PCIe AIR runtime requires the use of the [AIR PCIe kernel driver](https://github.com/Xilinx/ROCm-air-platforms/tree/main/driver). The driver directory in the [ROCm-air-platforms](https://github.com/Xilinx/ROCm-air-platforms) repository contains documentation on how to compile and load the AIR PCIe kernel driver.
```
Comment (jbelot) [inline docs/Building.md:118]: The second sentence seems not very useful as it points to the same directory. I'd say "This directory contains ...".

File: docs/Building.md [hunk 2]
```diff
@@ -118,42 +115,42 @@ and llvm.
...
 ```
 ./utils/clone-rocm-air-platforms.sh
 ```
 
-Then, set `${ROCM_ROOT}` to the ROCm install from the previous path. Then, run the following command to build the mlir-aie toolchain targetting the VCK5000.
+Then, set `${ROCM_ROOT}` to the ROCm install. Then, run the following command to build the mlir-aie toolchain targetting the VCK5000 pointing to the externally installed aie-rt and experimental ROCm runtime.
 
 ```
-./utils/build-mlir-aie-pcie.sh llvm/build/ build install /opt/xaienginev2 ${ROCM_ROOT}/lib/cmake/hsa-runtime64/ ${ROCM_ROOT}/lib/cmake/hsakmt/
+./utils/build-mlirgaie-pcie.sh llvm/build/ build install /opt/xaiengine ${ROCM_ROOT}/lib/cmake/hsa-runtime64/ ${ROCM_ROOT}/lib/cmake/hsakmt/
```
Comment (jbelot) [inline docs/Building.md:151]: I guess there is a typo here in `./utils/build-mlirgaie-pcie.sh` => `./utils/build-mlir-aie-pcie.sh`.

**Review by jbelot — CHANGES_REQUESTED** [review]

File: docs/Building.md (hunk 2 above)
Comment (eddierichter-amd) [reply docs/Building.md]: Thanks for catching! Addressed in [4ce47c8](https://github.com/Xilinx/mlir-aie/pull/1597/commits/4ce47c857d36eb4a344600c0b16b175ee7792ffc).

File: docs/Building.md (hunk 1 above)
Comment (eddierichter-amd) [reply docs/Building.md]: Thanks for catching! Addressed in [4ce47c8](https://github.com/Xilinx/mlir-aie/pull/1597/commits/4ce47c857d36eb4a344600c0b16b175ee7792ffc).

**Pull request merged**